source/appModules/eclipse.py

```diff
@@ -17,8 +25,70 @@ def event_valueChange(self):
 		# Therefore, just drop this event.
 		pass
 
+class AutocompletionListView(IAccessible):
+
+	def _get_name(self):
+		return AUTOCOMPLETION_LIST_NAME
+
+	def event_hide(self):
+		# NVDA filters the hide events, even requesting them, so it doesn't work:
+		eventHandler.executeEvent("gainFocus", api.getDesktopObject().objectWithFocus())
+
+class AutocompletionListItem(IAccessible):
+
+	def event_selection(self):
+		# Fixme: I don't know if it is the correct way to do this.

```

Comment:
This comment should be removed. If you have some concrete doubts, or cases you know are not handled add a comment to that effect.

Reply:
Hello,

I don't know how to write unit tests for an app module, I didn't read about it yet.

I've downloaded eclipse to test it again.